package/gluon-status-page/javascript/status-page.js

```diff
@@ -210,7 +210,8 @@
 
 			var td = document.createElement('td');
 			if (peer[1] && peer[1].established != null)
-				td.textContent = _['connected'] + ' (' + formats.time(peer[1].established) + ')';
+				const method = peer[1].method ? ', ' + peer[1].method : '';
```

Comment:
So far we avoid ES6+ features in our code. I think it would be fine to allow these features by now, but it should be a deliberate decision to change this policy.

Reply:
Haven't even thought about that, given that const is so widely supported and it is the ring thing to use in this scenario iirc.

https://caniuse.com/const